specs/core/0_beacon-chain.md

```diff
@@ -189,24 +190,36 @@ Code snippets appearing in `this style` are to be interpreted as Python code. Be
 | Name | Value |
 | - | - |
 | `GENESIS_FORK_VERSION` | `0` |
-| `GENESIS_SLOT` | `0` |
+| `GENESIS_SLOT` | `2**19` |
+| `GENESIS_EPOCH` | `slot_to_epoch(GENESIS_SLOT)` |
```

Comment:
Note that `slot_to_epoch` use `EPOCH_LENGTH` that is defined in the next section. So in the chain config, this will look like:
```python
epoch_length = 2**6
SERENITY_CONFIG = BeaconConfig(
    ...
    GENESIS_EPOCH=slot_to_epoch(GENESIS_SLOT, epoch_length)
    ...
    EPOCH_LENGTH=epoch_length
    ...
)
```

Reply:
Not the prettiest. I think we want to move to #504 which will use chain start time to seed the initial state rather than config.